# Overview

This file is going to act as an overview of the project, what I will be doing, and what goals it has.

## Software used for development:

#### Git:

Versions will be managed using git version control. Most likely, this will just contain lots of commits and pushes.

#### Github:

Github will be a place to store projects, code used for analysis, and any documentation. It will not be used for storage of any genomic data, which will use the shared lab storage.



## Primary goal

#### Easily download genomic data onto the shared drive

Make interacting with genomic data using genbank easier. Either make it so you can use a text file for downloading project data, or potentially a very simple gui.

Check for updates of genomic data-sets

Potentially keep genomic data in a central folder to make finding things easier

#### Dartmouth DAC pipeline integration

Be able to produce normalized cpms of all bulk RNAseq analysis. The pipeline from the Data Analytics core is very useful, potentially setting it up could be automated so that I would not need to set it up each time someone wants to perform bulk RNAseq analysis. I see a lot of situations that would make this difficult, so it might not be doable.

#### Have ways to view data

Need to automate creation of visuals or certain analyses. For example gene set enrichment / pathway analysis, volcano plots, etc. Once I get to this point, ask people in the lab what type of analyses they'd like to have automated.

## Secondary Goals

These goals are not necessary to the main goal, but would be very useful
